app/models/chargeback/consumption_with_rollups.rb

```diff
@@ -22,7 +22,7 @@ def hash_features_affecting_rate
 
     def tag_names
       @tag_names ||= @rollups.inject([]) do |memo, rollup|
-        memo |= rollup.tag_names.split('|') if rollup.tag_names.present?
+        memo |= rollup.resource.tags.collect(&:name) if rollup.resource.tags.collect(&:name).present?
```

Comment:
I think I understand this... @lpichler is this capturing the current tag names associated with the resource instead of using the tag names at the time the rollup was created?